t/TouchActions.t

```diff
@@ -31,11 +32,11 @@ my $ta = Appium::TouchActions->new(
 
 TOUCH_ACTIONS: {
     my ($x, $y) = (0.2, 0.2);
-    $ta->tap( $x, $y );
+    $ta->tap( $x, $y )->perform();
 
-    ok(exists $actions->{'mobile: tap'}, 'we send the correct javascript for precise taps');
-    is($actions->{'mobile: tap'}->{x}, $x, 'with the correct x coords');
-    is($actions->{'mobile: tap'}->{y}, $y, 'with the correct y coords');
+    ok(exists $actions->{'action'}->{'tap'}, 'we perform precise taps');
+    is($actions->{'options'}->{x}, $x, 'with the correct x coords');
+    is($actions->{'options'}->{y}, $y, 'with the correct y coords');
```

Comment:
Are you planning to add tests that hit the rest of the subroutines that you added to TouchActions.pm?